### PR TITLE
Add the svc and vsc modifications in the voltage init modification

### PR DESCRIPTION
**Pull request by FranckLecuyer**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java [hunk 1]
```diff
@@ -84,68 +95,92 @@ private void applyTransformerModification(Network network, Reporter subReporter)
...
+    private void applyStaticVarCompensatorModification(Network network, Reporter subReporter) {
+        voltageInitModificationInfos.getStaticVarCompensators().forEach(s -> {
+            StaticVarCompensator staticVarCompensator = network.getStaticVarCompensator(s.getStaticVarCompensatorId());
+            if (staticVarCompensator == null) {
+                Reporter reporter = subReporter.createSubReporter(STATIC_VAR_COMPENSATOR_MSG + s.getStaticVarCompensatorId(), STATIC_VAR_COMPENSATOR_MSG + s.getStaticVarCompensatorId());
+                report(reporter, "staticVarCompensatorNotFound", "Static var compensator with id=${id} not found", Map.of("id", s.getStaticVarCompensatorId()), TypedValue.WARN_SEVERITY);
+                return;
+            }
+            if (s.getVoltageSetpoint() != null || s.getReactivePowerSetpoint() != null) {
+                Reporter reporter = subReporter.createSubReporter(STATIC_VAR_COMPENSATOR_MSG + s.getStaticVarCompensatorId(), GENERATOR_MSG + s.getStaticVarCompensatorId());
```
Comment (dbraquart) [inline src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java:146]:
```suggestion
                Reporter reporter = subReporter.createSubReporter(STATIC_VAR_COMPENSATOR_MSG + s.getStaticVarCompensatorId(), STATIC_VAR_COMPENSATOR_MSG + s.getStaticVarCompensatorId());
```

**Review by dbraquart — CHANGES_REQUESTED** [review]
code: 1 minor change request
tests: ok

File: src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/modifications/VoltageInitModification.java]: Fixed

**Review by dbraquart — COMMENTED** [review]
code: ok
tests: ok

**Review by dbraquart — APPROVED** [review]
code: ok
tests: ok

File: src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitStaticVarCompensatorModificationEmbeddable.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
```
Comment (ayolab) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitStaticVarCompensatorModificationEmbeddable.java:2]:
```suggestion
 * Copyright (c) 2023, RTE (http://www.rte-france.com)
```

File: src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitVscConverterStationModificationEmbeddable.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
```
Comment (ayolab) [inline src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitVscConverterStationModificationEmbeddable.java:2]:
```suggestion
 * Copyright (c) 2023, RTE (http://www.rte-france.com)
```

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 4]
```diff
@@ -180,6 +180,11 @@ databaseChangeLog:
   - include:
       file: changesets/changelog_20230807T095308Z.xml
       relativeToChangelogFile: true
+  - include:
+      file: changesets/changelog_20230809T122316Z.xml
+      relativeToChangelogFile: true
   - include:
       file: changesets/changelog_20230810T134233Z.xml
       relativeToChangelogFile: true
+
```
Comment (ayolab) [inline src/main/resources/db/changelog/db.changelog-master.yaml:189]: remove white spaces

**Review by ayolab — APPROVED** [review]

File: src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitStaticVarCompensatorModificationEmbeddable.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/entities/equipment/modification/VoltageInitStaticVarCompensatorModificationEmbeddable.java]: Fixed

File: src/main/resources/db/changelog/db.changelog-master.yaml (hunk 4 above)
Comment (FranckLecuyer) [reply src/main/resources/db/changelog/db.changelog-master.yaml]: Fixed

**Pull request merged**